meta: Update changelog for 23.02rc3

endothermicdev opened the pull request:
Changelog-None

vincenzopalazzo reviewed (commented):
we should add here the recent fixes?

endothermicdev commented:
With rc3, I don't think we fixed anything that was a bug in 22.11, but correct me if I'm wrong.  We started sending an even feature bit in rc1 and we added node_announcements that shouldn't have been there (and crashed gossmap) in rc1.  Those have been addressed, but those weren't bugs in a previous official release so the don't get documented in changelog, just in the commit message.

endothermicdev commented:
ACK 6009f8dbd0ecf505a9a24e0b7995fe4a8c5e1ec1
